app/vlagent/kubernetescollector/collector.go

```diff
@@ -12,9 +12,10 @@ import (
 	"sync"
 	"time"
 
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
+
 	"github.com/VictoriaMetrics/VictoriaLogs/app/vlagent/remotewrite"
 	"github.com/VictoriaMetrics/VictoriaLogs/lib/logstorage"
-	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
```

Comment:
Please return back this import to the previous location. The imports must be divided into three groups sorted by import paths:

1. Standard Go packages
2. External packages
3. Packages from the current repository

The `github.com/VictoriaMetrics/VictoriaMetrics/lib/logger` is an external package, so it must belong to the second group.

Reply:
Pushed a commit that sorts imports into three groups in other files in this package, as you specified in your message